unlock-app/src/components/creator/CreatorLog.tsx

```diff
@@ -1,34 +1,64 @@
 import React from 'react'
 import styled from 'styled-components'
+import { connect } from 'react-redux'
 import * as UnlockTypes from '../../unlockTypes'
+import { DefaultError } from './FatalError'
+import Loading from '../interface/Loading'
 
 interface Props {
   transactionFeed: UnlockTypes.Transaction[]
   explorerLinks: { [key: string]: string }
+  loading: boolean
 }
 
-const CreatorLog = ({ transactionFeed, explorerLinks }: Props) => {
+const CreatorLog = ({ transactionFeed, explorerLinks, loading }: Props) => {
   return (
-    <Grid>
-      <HeaderItem>Block Number</HeaderItem>
-      <HeaderItem>Lock Name/Address</HeaderItem>
-      <HeaderItem>Type</HeaderItem>
-      {transactionFeed.map(tx => {
-        return (
-          <React.Fragment key={tx.hash}>
-            <BlockNumber>{tx.blockNumber}</BlockNumber>
-            <Address href={explorerLinks[tx.hash]} target="_blank">
-              {tx.lock}
-            </Address>
-            <Type type={tx.type}>{tx.type}</Type>
-          </React.Fragment>
-        )
-      })}
-    </Grid>
+    <>
```

Comment:
Nit: do you mind using `<Fragment>` here? We found that it is slightly less confusing.